packages/dependency-profiles/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@fluentui-react-native/dependency-profiles",
+  "version": "0.0.0",
```

Comment:
Is this supposed to be version 0?

Reply:
Yeah, I've seen new packages start at `0.0.0` or `0.1.0`. I don't think it matters because Beachball will bump it to a reasonable number as soon as it gets a chance to run (after this PR is merged).